src/IO/S3/copyS3File.h

```diff
@@ -21,7 +21,7 @@ class SeekableReadBuffer;
 /// however copyS3File() is faster and spends less network traffic and memory.
 /// The parameters `src_offset` and `src_size` specify a part in the source to copy.
 void copyS3File(
-    const std::shared_ptr<const Aws::S3::S3Client> & s3_client,
+    const std::shared_ptr<const S3::S3Client> & s3_client,
```

Comment:
There are too many `S3` in `S3::S3Client` :)

Reply:
Now that you say it, I agree, we could leave with `S3::Client`. 😄